pkg/controllers/disruption/drift.go

```diff
@@ -82,16 +82,35 @@ func (d *Drift) ComputeCommand(ctx context.Context, candidates ...*Candidate) (C
 		consolidationTypeLabel: d.ConsolidationType(),
 	}).Set(float64(len(candidates)))
 
+	// Do a quick check through the candidates to see if they're empty.
+	// For each candidate that is empty with a nodePool allowing its disruption
+	// add it to the existing command.
+	empty := make([]*Candidate, 0, len(candidates))
+	for _, candidate := range candidates {
+		if len(candidate.pods) > 0 {
```

Comment:
It feels like this emptiness function can be refactor out? It looks the same in all the controllers

Reply:
Agreed. Good callout.